project/Dependencies.scala

```diff
@@ -1,14 +1,14 @@
 import sbt._
 
 object Dependencies {
-  val akkaV = "2.5.22"
-  val akkaHttpV = "10.1.8"
+  val akkaV = "2.6.1"
+  val akkaHttpV = "10.1.11"
   val jacksonV = "2.9.5"
   val scalaLoggingV = "3.5.0"
   val scalaTestV    = "3.0.5"
   val scalaCheckV    = "1.14.0"
   val catsEffectV         = "2.1.1"
-  val scalikejdbcVersion    = "3.3.5"
+  val scalikejdbcVersion    = "3.4.1"
```

Comment:
Fix `NoClassDefFoundError: scala/collection/compat/Factory` issue. https://github.com/scalikejdbc/scalikejdbc/issues/983

  /*
root[ERROR] Exception in thread "pool-2-thread-9" java.lang.NoClassDefFoundError: scala/collection/compat/Factory$
root[ERROR] 	at scalikejdbc.DBSession.list(DBSession.scala:295)
   */